hack/make/.build-deb/control

```diff
@@ -7,6 +7,7 @@ Vcs-Git: git://github.com/docker/docker.git
 Package: docker-engine
 Architecture: linux-any
 Depends: iptables, ${misc:Depends}, ${perl:Depends}, ${shlibs:Depends}
+Build-Depends: debhelper, dh-apparmor
```

Comment:
except if we distributed source debs, this *would* be necessary for someone to build from them, yes?